setup.py

```diff
@@ -1,21 +1,54 @@
+#!/usr/bin/env python
+
+import os.path
+import re
+from setuptools import setup, find_packages
+
+
+curdir = os.path.dirname(os.path.realpath(__file__))
+filename = os.path.join(curdir, 'ipymd', '__init__.py')
+with open(filename, 'r') as f:
+    version = re.search(r"__version__ = '([^']+)'", f.read()).group(1)
+
+classes = """
+    Development Status :: 3 - Alpha
+    License :: OSI Approved :: BSD License
+    Environment :: Console
+    Framework :: IPython
+    Intended Audience :: Developers
+    Natural Language :: English
+    Operating System :: Unix
+    Operating System :: POSIX
+    Operating System :: MacOS :: MacOS X
+    Programming Language :: Python
+    Programming Language :: Python :: 2
+    Programming Language :: Python :: 2.7
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.3
+    Programming Language :: Python :: 3.4
+    Topic :: Software Development :: Libraries
+    Topic :: Text Processing :: Markup
+
 """
-ipymd
-"""
-from setuptools import setup
-
-setup(
-    name='ipymd',
-    version='0.1.0-dev',
-    packages=['ipymd',
-              'ipymd.core',
-              'ipymd.ext',
-              'ipymd.formats',
-              'ipymd.lib',
-              'ipymd.utils',
-              ],
-    entry_points={
-        'console_scripts': [
-            'ipymd=ipymd.core.scripts:main',
-            ]
-        }
-    )
+classifiers = [s.strip() for s in classes.split('\n') if s]
+
+description = ('Use the IPython notebook as an interactive Markdown editor')
+
+with open('README.md') as f:
+    long_description = f.read()
+
+setup(name='ipymd',
+      version=version,
+      license='BSD',
+      description=description,
+      long_description=long_description,
+      author='Cyrille Rossant',
+      author_email='cyrille.rossant at gmail.com',
+      maintainer='Cyrille Rossant',
+      maintainer_email='cyrille.rossant at gmail.com',
+      url='https://github.com/rossant/ipymd',
+      classifiers=classifiers,
+      packages=find_packages(),
+      entry_points={'console_scripts': ['ipymd=ipymd.core.scripts:main']},
+      install_requires=['ipython[notebook] >= 3.0'],

```

Comment:
We have not been able to reproduce this error on our end, though we were seeing it  with travis prior to changing the installation method in the `.travis.yml` to use pip. Can you tell me where you are seeing the error now and we'll trouble shoot that?

I can take this line out of `setup.py` however I believe this will cause the tests to fail and therefore travis. Another  possibility would be to include the necessary dependencies for the tests individually (such as tornado and jinja2), I don't believe there are too many extra dependencies.